teamsupport/models.py

```diff
@@ -7,15 +7,21 @@
 from teamsupport.errors import MissingArgumentError
 
 
-class XmlModel(object):
+def _find_dict_by_key_value(list_of_dicts, **kwargs):
+    for dct in list_of_dicts:
+        for (key, value) in kwargs.items():
+            if dct.get(key) == value:
+                return dct
```

Comment:
It looks like this returns the first time a kwarg matches, not only if all kwargs match.  Is that the intended behavior?  If it should be matching all kwargs, this might be a good place to use an `else` clause on the `for (key, value) in kwargs.items():` loop.
It is accepts multiple kwargs maybe it should be renamed to indicate it matches on multiple items?
`_find_dict_with_items`?
`_find_dict_with_key_value_pairs`?
`_find_dict_matching_items`?
`_find_dict_matching_key_value_pairs`?